lib/pure/cookies.nim

```diff
@@ -9,18 +9,23 @@
 
 ## This module implements helper procs for parsing Cookies.
 
-import strtabs, times
+import std/[strtabs, times, options]
+
+
+type
+  SameSite* {.pure.} = enum ## The SameSite cookie attribute.
+    None, Lax, Strict
 
 proc parseCookies*(s: string): StringTableRef =
```

Comment:
is `StringTableRef` even meaningful?

`nim r --eval:'import std/[cookies,strtabs]; echo parseCookies("a=1;Path=/; a=2;Path=/example;")'` seems wrong according to https://stackoverflow.com/questions/4056306/how-to-handle-multiple-cookies-with-the-same-name

Reply:
=> moved this discussion to https://github.com/timotheecour/Nim/issues/602#issuecomment-782600451